python/overlay_widget/shotgun_overlay_widget.py

```diff
@@ -45,16 +55,18 @@ def __init__(self, parent):
 
         self._shotgun_spinning_widget = ShotgunSpinningWidget(self)
 
-        # make it transparent
-        self.setAttribute(QtCore.Qt.WA_TransparentForMouseEvents)
+        # We want text to be centered and wrapping words.
+        self.setAlignment(QtCore.Qt.AlignCenter | QtCore.Qt.AlignVCenter | QtCore.Qt.TextWordWrap)
+        self.setWordWrap(True)
 
-        # turn off the widget
-        self.setVisible(False)
-        self._mode = self.MODE_OFF
+        # Allow to open hyperlinks
+        self.setOpenExternalLinks(True)
+
+        # Dark gray background.
+        self.setStyleSheet("background-color: #1B1B1B")
```

Comment:
Not to add too much more work, but I think the convention would be to add a `style.qss` file into this widget's folder and apply it when the widget is created. This avoids hardcoding the style in the code.

Reply:
You might be able to set the label text with div or span tags and use class selectors in the qss file to handle the error vs. info colors. i.e.: `label.setText("<div className='error'>Error message</div>")` ... then, in the qss file:

```
.error {
    color: red;
}
```

I haven't tried this before actually so not sure if it will work.

Reply:
I've looked at the Qt doc and it doesn't seem like you can specify contants in that file. You can only have selectors.